flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcRowDataLookupFunction.java

```diff
@@ -56,13 +57,16 @@ public class JdbcRowDataLookupFunction extends LookupFunction {
     private static final Logger LOG = LoggerFactory.getLogger(JdbcRowDataLookupFunction.class);
     private static final long serialVersionUID = 2L;
 
-    private final String query;
+    private String query;
     private final JdbcConnectionProvider connectionProvider;
-    private final String[] keyNames;
+    private String[] keyNames;
     private final int maxRetryTimes;
     private final JdbcRowConverter jdbcRowConverter;
     private final JdbcRowConverter lookupKeyRowConverter;
 
+    private List<String> resolvedPredicates = new ArrayList<>();
+    private Serializable[] pushdownParams = new Serializable[0];
```

Comment:
We do not need this initialization here since it is not used anywhere and then is going to be overwritten in a constructor.
After that both vars could be made final